msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1120,6 +1121,23 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`alternateIcons.highResolutionIcon.size`| Number enum | | ✔️ | This property is reserved for future use. The value must be 64 (not 128). |
 |`alternateIcons.highResolutionIcon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in on high DPI screens. Icon image must be 128 x 128 pixels and use one of the following file formats: GIF, JPG, PNG, EXIF, BMP, TIFF.|
 
+### extensions.contextMenus
+
+A context menu is a shortcut menu that appears when you select and hold (or right-click) in the Office UI.
```

Comment:
Same for this instance. Just want to disambiguate from "Microsoft 365 (Office)" app (Harmony)